pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -240,6 +240,38 @@ func (b *OFBridge) DeleteMeterAll() error {
 	return nil
 }
 
+func (b *OFBridge) GetMeterStats(handleMeterStatsReply func(packetCounts map[int]int64)) error {
+	const OFPM_ALL = 0xffffffff // Represents all meters
+	mpMeterStatsReq :=
+		openflow15.NewMpRequest(openflow15.MultipartType_MeterStats)
+	meterMPReq := openflow15.NewMeterMultipartRequest(OFPM_ALL)
+	mpMeterStatsReq.Body = append(mpMeterStatsReq.Body, meterMPReq)
+	ch := make(chan *openflow15.MultipartReply, 1)
+	b.multipartReplyChs[mpMeterStatsReq.Xid] = ch
+	go func() {
+		defer func() {
+			delete(b.multipartReplyChs, mpMeterStatsReq.Xid)
+		}()
+		packetCounts := make(map[int]int64)
+		select {
+		case reply := <-ch:
+			if reply.Type == openflow15.MultipartType_MeterStats {
+				statsArr := reply.Body
```

Comment:
maybe `statsResponse`? or you may replace `statsArr` in following line with `reply.Body` directly since I didn't see `statsArr` being used anywhere else.

Reply:
Updated accordingly, thanks!